src/components/UserInfoHeader/withData.tsx

```diff
@@ -16,6 +16,9 @@ interface UserInfoWithDataPropType {
   activeTab: "sensors" | "tokens";
 }
 
+export const getRandomSensorId = (): number =>
+  Math.round(Math.random() * (32 - 1) + 1);
```

Comment:
Is there a way we can make this `32` less magic? Maybe assign it to a variable such as `NUMBER_OF_SYMBOL_SVGS`?